alerts/openshift-container-storage-operator/CephMdsCpuUsageHigh.md

```diff
@@ -0,0 +1,33 @@
+# MDSCacheUsageHigh
+
+## Meaning
+
+Ceph MDS cpu usage for the MDS daemon has exceeded the thereshold for adaquete performance.
+
+## Impact
+
+MDS serves filesystem metadata. The MDS is crucial for any file creation, rename, deletion and update operations. When the load is more than the CPU resource for the MDS, we get this alert. High CPU usage means the MDS performance deteriorates and may cause MDS to crash.
+
+## Diagnosis
+
+MDS is by default is allocated 2 or 3 CPU. It is okay as long as metadata operations are not too many. When the metadata operation load increases enough to trigger this alert, it means the default CPU allocation is unable to cope with load and we need to increase the CPU allocation or run multiple active metadata servers.
```

Comment:
This is an excellent explanation to be moved to "impact".
In the "diagnosis" section you should explain the user how to verify or "double check" the situation warned by the alert.
Suggestion:
Explain how to use the "observe" tab to display the MDS cpu usage, and to verify that CPU usage is above the alert threshold.

Reply:
ack.